src/components/static/Navigation.svelte

```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
```

Comment:
Removing `justify-center` makes it off centered, is this the expected behavior?

Reply:
Updated this to justify-left to align it with "Product section needs to be aligned with MCSWf logo it is not."